config/locales/responders.en.yml

```diff
@@ -0,0 +1,12 @@
+en:
+  flash:
+     families:
+       create:
+         notice: '%{resource_name} was successfully created.'
+         alert: '%{resource_name} could not be created.'
+       update:
+         notice: '%{resource_name} was successfully updated.'
+         alert: '%{resource_name} could not be updated.'
+       destroy:
+         notice: '%{resource_name} was successfully destroyed.'
+         alert: '%{resource_name} could not be destroyed.'
```

Comment:
I'm not sure whether you should have included the destroy notice and alert yet as you don't yet have it in your controller and routes?

Reply:
The file has been created automatically with the responders inplementation. I will need this so I think there is not point to delete it to add it again late. I think! ;) But good point!

Reply:
No, sorry didn't realise that, I would leave it there too!